client/src/styles/legacy/_tree.scss

```diff
@@ -493,6 +493,22 @@
     ins {
       background-color: transparent;
       background-image: url(../images/sitetree_ss_default_icons.png);
+
+      &.font-icon-drag-handle {
+        background: none;
+        font-size: 1.15385rem; // 15px
```

Comment:
Is there value here in using `rem` for `.font-icon-drag-handle` and `.font-icon-right-dir`?

Looks like we are clearly targeting a pixel size. And that those size are meant to line up with margins else where in the file.